FSW/src/headers/config.hpp

```diff
@@ -106,6 +97,11 @@ const int COMMAND_QUEUE_SIZE = 100;     // maximum number of commands the comman
 
 
 /* - - - - - - Fault Mitigation Module - - - - - - */
+static volatile bool ACT_ON_NEW_FAULTS = true;
+const int PERSIST_DATA_ADDR = 0; // first address of persistent system data in EEPROM
+const uint8_t EXPECTING_RESTART_FLAG = 0xaa; // 10101010, value of flag indicating that the last restart was expected.

```

Comment:
I'm guessing capitalization doesn't matter on this hex value?

Reply:
Yep